packages/block-editor/src/store/defaults.js

```diff
@@ -172,9 +172,17 @@ export const SETTINGS_DEFAULTS = {
 
 	// This setting is `private` now with `lock` API.
 	blockInspectorAnimation: {
+		animationParent: 'core/navigation',
```

Comment:
I haven't dug deep into `blockInspectorAnimation` API. Is it an API that we will reuse in other places and blocks? If yes, `animationParent`(or whatever the final API will be), should be inside each block, like `enterDirection` is. If not, maybe it's something to be implemented differently and more close to Navigation block? 🤔

Reply:
It's only used for the navigation block at the moment. I am aware that this implementation won't scale well, but since it's locked I wasn't too worried about that. We can improve it when we need it to do more.

Reply:
If this API is tightly coupled with Navigation block, have you explored a way to handle this 'closer' to the block? Also you have explored and still think it should be here, should we rename to reflect that? `blockInspectorAnimation` is definitely a generic name, that may create confusion I think..

Reply:
This APi should work for all blocks that have children which are controlled by the parent in various ways. For now the two examples are navigation block and the pattern block in content only mode.

It is not an API specific to the navigation block and it should not be at the block level.

@ntsekouras which part do you think should be extracted to a hook?